src/ui/layout/mct-tree.vue

```diff
@@ -189,21 +187,30 @@ export default {
             return {
                 paddingLeft: offset + 'px'
             };
+        },
+        ancestorsHeight() {
+            if (this.activeSearch) {
+                return 0;
+            }
+
+            return this.itemHeight * this.ancestors.length;
+        },
+        pageThreshold() {
+            return Math.ceil(this.availableContainerHeight / this.itemHeight) + ITEM_BUFFER;
+        },
+        scrollActive() {
+            return !(this.childrenHeight > this.availableContainerHeight);
```

Comment:
`childrenHeight` has been re-implemented as a method, so this check won't work any more.

Reply:
Well that was enlightening! Looks like at this point I didn't even need the scrollActive variable.